packages/ipfs-topology/src/ipfs-topology.ts

```diff
@@ -118,41 +81,32 @@ export class IpfsTopology {
     }
   }
 
-  private async _dynamicBoostrapList(network: string): Promise<Multiaddr[]> {
-    const url = PEER_FILE_URLS(network as Networks)
-    if (!url) {
-      return []
-    }
-    this.logger.imp(`Connecting to peers found in '${url}'`)
-    const list = await fetchJson(url)
-    return list?.map((peer) => new Multiaddr(peer)) || []
-  }
-
   private async _forceBootstrapConnection(
     ipfs: IpfsApi,
     bootstrapList: Multiaddr[]
   ): Promise<void> {
     // Don't want to swarm connect to ourself
-    const myPeerId = (await ipfs.id()).id
+    let myPeerId
+    try {
+      myPeerId = (await ipfs.id()).id
+    } catch (error) {
+      this.logger.warn(
+        `Error loading our PeerID from IPFS: ${error}. Skipping connection to bootstrap peers`
+      )
+      return
+    }
+
     const filteredBootstrapList = bootstrapList.filter((addr) => {
       return !addr.getPeerId()?.endsWith(myPeerId)
     })
 
-    await Promise.all(
-      filteredBootstrapList.map(async (node) => {
-        try {
-          await ipfs.swarm.connect(node)
-        } catch (error) {
-          this.logger.warn(`Can not connect to ${node}`)
-          this.logger.warn(error)
-        }
-      })
-    )
-
-    const connectedPeers = await ipfs.swarm.peers()
-    if (connectedPeers.length > 0) {
-      const peerAddresses = connectedPeers.map((obj) => obj.addr)
-      this.logger.imp(`Connected to peers: ${peerAddresses.join(', ')}`)
+    for (const node of filteredBootstrapList) {
+      try {
+        await ipfs.swarm.connect(node)
+      } catch (error) {
+        this.logger.warn(`Can not connect to ${node}`)
+        this.logger.warn(error)
+      }
```

Comment:
Why is this code changed from an async map to a sync for loop?

Reply:
so it happens one at a time instead of hitting the IPFS node with a bunch of requests all at once.  Trying to minimize load on IPFS